### ISSUE TITLE
We need a logo for the project

### ISSUE THREAD

**Issue by mrkn** [yoshoku/rumale#4]
I think it would be nice if this project has a fashionable logo.

**Comment by yoshoku** [comment]
I will try asking that to my friend who is a designer 👍

**Comment by yoshoku** [comment]
The Rumale logos are complete. We have created two types of logos, icon and header type. 

![rumale_logo1](https://user-images.githubusercontent.com/5562409/56179537-0ab0a400-6041-11e9-9bf5-1fc9e813ed45.png)
![rumale_logo2](https://user-images.githubusercontent.com/5562409/56179541-0d12fe00-6041-11e9-8aad-7cea23eadecc.png)

We adopted an abstract bird in the logo motif with reference to [the ugly duckling theorem (Satosi Watanabe, 1969)](https://en.wikipedia.org/wiki/Ugly_duckling_theorem), a famours theorem for machine learning and pattern recognition. I am mainly going to use the icon type. I will add the logo to READM.md as soon as possible.
